tasks/recover/register_template.yml

```diff
@@ -0,0 +1,31 @@
+- block:
+  - name: Register unregistered Template
+    ovirt_templates:
+        state: registered
+        storage_domain: "{{ storage.name }}"
+        id: "{{ unreg_template.id }}"
+        allow_partial_import: "{{ dr_partial_import }}"
+        auth: "{{ ovirt_auth }}"
+        cluster_mappings: "{{ dr_cluster_map }}"
+        domain_mappings: "{{ dr_domain_map }}"
+        vnic_profile_mappings:  "{{ dr_network_map }}"
+        role_mappings: "{{ dr_role_map }}"
+
+        # TODO: We should set a flag
+        # - fail: msg="The execution has failed because of errors."
+        #  when: flag == "failed"
+        #
+        #  - name: Set flag
+        #  set_fact: flag = failed
+        #  when: "'FAILED' in command_result.stderr"
+    register: template_register_result
+
+  - name: Report issues
```

Comment:
please indicate it is for templates, like this: "Append failed template to issues report"